pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1494,11 +1494,29 @@ cdef class BusinessDay(BusinessMixin):
     """
     DateOffset subclass representing possibly n business days.
 
+    Parameters
+    ----------
+    n : int, default 1
+        The number of days represented.
+    normalize : bool, default False
+        Normalize start/end dates to midnight before generating date range.
```

Comment:
Can this be used outside of generating a date range? If so, maybe

    Normalize start/end dates to midnight

is enough?

Reply:
Yes, I think so. Changed this line.

Thank you for your comments.  I created a new commit.